Jenkinsfile.deploy

```diff
@@ -6,6 +6,12 @@ def notify(message, color='good') {
     }
 }
 
+def delay() {
```

Comment:
You should place a comment here stating why this is done. I can see future me finding this and getting indignant as to why someone would put a delay in the build.